litert/vendors/qualcomm/compiler/qnn_compose_graph.cc

```diff
@@ -989,8 +989,22 @@ void AddTensorToQnn(
     if (use_qint16_as_quint16) {
       tensor.ConvertQint16ToQuint16();
     }
-    qnn_api->tensorCreateGraphTensor(graph_handle, &(tensor.GetQnnTensor()));
-    created_tensors.emplace(&tensor);
+    auto error = qnn_api->tensorCreateGraphTensor(graph_handle,
+                                                  &(tensor.GetQnnTensor()));
+    if (QNN_SUCCESS != error) {
+      const char* message = nullptr;
+      auto get_message_error = qnn_api->errorGetMessage(error, &message);
+      if (QNN_SUCCESS == get_message_error) {
+        LITERT_LOG(LITERT_ERROR,
+                   "failed to create graph tensor, error: %d, message: %s",
+                   error, message);
+      } else {
+        LITERT_LOG(LITERT_ERROR, "failed to create graph tensor, error: %d",
+                   error);
+      }
```

Comment:
Open for discussion, do you think we can directly print message(`nullptr`) even we failed to get error message?

Reply:
Maybe we can change to:
```suggestion
      if (QNN_SUCCESS == get_message_error) {
        LITERT_LOG(LITERT_ERROR,
                   "failed to create graph tensor, error: %d, message: %s",
                   error, message);
      } else {
        LITERT_LOG(LITERT_ERROR, "failed to create graph tensor and error message, error: %d",
                   error);
      }
```